### PR TITLE
cli: add flag to list output URLs

### LINKED ISSUE #297: cli: add option to list workflow output files as URL's
As an enrichment of the `ls` command, we could offer users the possibility to list the files as URLs:

```console
$ reana-client ls --url
https://reana.cern.ch/api/workflows/{workflow_id_or_name}/workspace/{file_name}
https://reana.cern.ch/api/workflows/{workflow_id_or_name}/workspace/{file_name}
https://reana.cern.ch/api/workflows/{workflow_id_or_name}/workspace/{file_name}
```

The same output could be added to `reana-client run --follow` output, showing the list of files in case of success.

### PR DESCRIPTION
**Pull request by leticiawanderley**
Add new flag to the list workspace files command.

Allow user to retrieve files' URLs

Closes #297

Signed-off-by: leticia <leticia.farias.wanderley@cern.ch>

### CODE REVIEW

File: reana_client/cli/files.py [hunk 1]
```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    JSON = 'json'
+    URL = 'url'
+    FILE_PATH = "https://reana.cern.ch/api/workflows/%s/workspace/%s"
```
Comment (leticiawanderley) [inline reana_client/cli/files.py]: Is this `reana.cern.ch` the only possible address? Or should we use the `current_rs_api_client.swagger_spec.api_url` value?

Comment (diegodelemos) [reply reana_client/cli/files.py]: To get the URL you can use [`ctx.obj`](https://github.com/diegodelemos/reana-client/blob/master/reana_client/cli/__init__.py#L87) which contains the configuration with the `REANA_SERVER_URL` set by the user: [`ctx.obj. reana_server_url`](https://github.com/diegodelemos/reana-client/blob/master/reana_client/cli/__init__.py#L29) :)

File: reana_client/cli/files.py [hunk 2]
```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    JSON = 'json'
+    URL = 'url'
```
Comment (diegodelemos) [inline reana_client/cli/files.py]: We might need more output formats later, what do you think about moving it to config?

File: reana_client/cli/files.py [hunk 3]
```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
...
+    FILE_PATH = "%s/api/workflows/%s/workspace/%s"
```
Comment (diegodelemos) [inline reana_client/cli/files.py]: If you would be able to retrieve [this string](https://github.com/reanahub/reana-commons/blob/master/reana_commons/openapi_specifications/reana_server.json#L1835) from the `api_client` you could do as follows (and we would be save from having this string outdated in the future 😄):

```python
In[1]: "/api/workflows/{workflow_id_or_name}/workspace/{file_name}".format(workflow_id_or_name='1234', file_name='test.json')                                                     
Out[2]: '/api/workflows/1234/workspace/test.json'
```

**Pull request merged**